tools/releaseTools.psm1

```diff
@@ -137,9 +137,12 @@ function New-CommitNode
 function Get-ChangeLog
 {
     param(
-        [Parameter(Mandatory)]
+        [Parameter(Mandatory = $true)]
         [string]$LastReleaseTag,
 
+        [Parameter(Mandatory = $false)]
```

Comment:
@adityapatwardhan 
I think we should make the mandatory?

Reply:
I agree, this should be mandatory

Reply:
```suggestion
        [Parameter(Mandatory)]
```